README

```diff
@@ -0,0 +1,41 @@
+# Ngrok Agent Docker Image
+
+## Usage
+
+Run an ngrok agent pointed at localhost:80
+
+```bash
+docker run ngrok/ngrok http 80
+```
+
+Access the web inspector on the host machine at localhost:3000
+
+```bash
+docker run --publish 3000:4040 ngrok/ngrok http 80
+```
+
+Run the ngrok agent with auth token 'xyz'
+
+```bash
+docker run -e NGROK_AUTHTOKEN=xyz ngrok/ngrok:alpine http 80
+```
+
+Run the ngrok agent with the config file './ngrok.yml' from the host machine
+
+```bash
+docker run -v $(pwd)/ngrok.yml:/etc/ngrok.yml -e NGROK_CONFIG=/etc/ngrok.yml ngrok/ngrok:alpine http 80
+```
+
+## Tagging
```

Comment:
Tagging -> Tags I think.

My vote is also to put the canonical list of tags in the dockerhub readme rather than here, and then just link to that readme.

This matches what the docker library images do, and I think it's quite slick.

Reply:
sounds good- is there anything else that should go in the dockerhub readme?

Reply:
Maybe an intro blurb. It should definitely link to ngrok.com and the docs on ngrok.com as well.